packages/medusa/src/workflows/functions/remove-products.ts

```diff
@@ -0,0 +1,12 @@
+import { MedusaContainer, ProductTypes } from "@medusajs/types"
```

Comment:
todo: Either the filename is wrong or the method body + name is. I am guessing the former :)

Reply:
The changes are part of the other pr herre https://github.com/medusajs/medusa/pull/4473